network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -202,6 +203,14 @@ def get_parent(self):
     def get_children(self):
         return BuyersGuideProductCategory.objects.filter(parent=self)
 
+    def get_ancestors(self, inclusive=False):
+        ancestors = []
+        node = self if inclusive else self.parent
+        while node:
+            ancestors.insert(0, node)
+            node = node.parent
+        return ancestors
+
```

Comment:
It feels like we shouldn't need this: given the level of nesting we're allowing atm, if a parent has a parent, that's a signal that something's going wrong already.

Reply:
I think that's answered [this question](https://github.com/mozilla/foundation.mozilla.org/issues/7644#issuecomment-989003970) :+1:

Reply:
ah, missed that comment. Replied, but for now let's go with depth 1.